### ISSUE TITLE
Replace cluster time-tree with Pangolin lineages

### ISSUE THREAD

**Issue by ArtPoon** [PoonLab/covizu#135]
In the old pipeline, we were using a hierarchical clustering analysis to partition the genome variants into clusters, and then building a time-scaled tree relating these clusters.

Now that we are adopting the Pangolin lineage nomenclature, we need to select representative genome sequences for each lineage and then build a time-scaled tree relating these entries to display in the left plot.

**Comment by ArtPoon** [comment]
We need a better method for selecting a representative genome for each Pangolin lineage.  Currently the code is using the sample with the earliest collection date - however, this happens to return a problematic genome for lineage B.1.11 (see below).  Accomplishing this requires knowing the number of differences from the reference genome for each genome, but we are not currently storing this information in the database.

```R
> require(ggfree)
Loading required package: ggfree
Loading required package: ape

Attaching package: ‘ggfree’

The following object is masked from ‘package:ape’:

    unroot

> phy <- read.tree('~/git/covizu/working/timetree.nwk')
> L <- tree.layout(phy)
Warning message:
In as.phyloData(phy, unscaled) :
  as.phyloData ignoring attribute root.edge of length 1
> plot(L, cex.lab=0.5)
```
![image](https://user-images.githubusercontent.com/1109328/95609243-69838d80-0a2c-11eb-8cd7-752d965c571f.png)

**Comment by ArtPoon** [comment]
This gets me what I need:
```sql
select header, unaligned from RAWSEQ where accession in (select accession from LINEAGE where lineage='B.1.6');
```

**Comment by ArtPoon** [comment]
Closed with commit 44230c0697ae570974035b9c1c03967f0d3c400e

**Comment by ArtPoon** [comment]
```console
art@orolo:~/git/covizu$ python3 covizu/treetime.py --ft2bin fasttree2-mp
FastTree Version 2.1.10 Double precision (No SSE3), OpenMP (32 threads)
Alignment: standard input
Nucleotide distances: Jukes-Cantor Joins: balanced Support: SH-like 1000
Search: Normal +NNI +SPR (2 rounds range 10) +ML-NNI opt-each=1
TopHits: 1.00*sqrtN close=default refresh=0.80
ML Model: Jukes-Cantor, CAT approximation with 20 rate categories
Ignored unknown character X (seen 11699 times)
Initial topology in 2.41 seconds0 of    162    1 of    165 seqs    100)   
Refining topology: 29 rounds ME-NNIs, 2 rounds ME-SPRs, 15 rounds ML-NNIs
      2.51 seconds: ME NNI round 1 of 29, 101 of 163 splits, 25 changes (max del      2.65 seconds: ME NNI round 2 of 29, 101 of 163 splits, 15 changes (max del      2.79 seconds: ME NNI round 3 of 29, 101 of 163 splits, 12 changes (max del      2.91 seconds: ME NNI round 4 of 29, 101 of 163 splits, 11 changes (max delTotal branch-length 0.015 after 12.03 sec, 1 of 163 splits   

WARNING! This alignment consists of closely-related and very-long sequences.
WARNING! FastTree (or other standard maximum-likelihood tools)
may not be appropriate for aligments of very closely-related sequences
like this one, as FastTree does not account for recombination or gene conversion

     15.41 seconds: ML NNI round 1 of 15, 101 of 163 splits, 38 changes (max delML-NNI round 1: LogLk = -46415.192 NNIs 66 max delta 0.00 Time 16.82
Switched to using 20 rate categories (CAT approximation)19 of 20   
Rate categories were divided by 0.627 so that average rate = 1.0
CAT-based log-likelihoods may not be comparable across runs
Use -gamma for approximate but comparable Gamma(20) log-likelihoods
     20.47 seconds: ML NNI round 2 of 15, 101 of 163 splits, 26 changes (max delML-NNI round 2: LogLk = -46258.849 NNIs 39 max delta 0.00 Time 21.74
Turning off heuristics for final round of ML NNIs (converged)
     23.61 seconds: ML NNI round 3 of 15, 101 of 163 splits, 14 changes (max delML-NNI round 3: LogLk = -46258.849 NNIs 27 max delta 0.00 Time 24.84 (final)
Optimize all lengths: LogLk = -46258.849 Time 25.90
Total time: 53.10 seconds Unique: 165/165 Bad splits: 0/162ernal splits   

Attempting to parse dates...
	Using column 'name' as name. This needs match the taxon names in the tree!!
	Using column 'date' as date.

0.00	-TreeAnc: set-up
/usr/local/lib/python3.6/dist-packages/phylo_treetime-0.7.5-py3.6.egg/treetime/aa_models.py:108: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray

2.08	WARNING: Previous versions of TreeTime (<0.7.0) RECONSTRUCTED sequences of
    	tips at positions with AMBIGUOUS bases. This resulted in unexpected
    	behavior is some cases and is no longer done by default. If you want to
    	replace those ambiguous sites with their most likely state, rerun with
    	`reconstruct_tip_states=True` or `--reconstruct-tip-states`.

3.19	TreeTime.reroot: with method or node: least-squares

3.19	TreeTime.reroot: rerooting will ignore covariance and shared ancestry.

3.41	TreeTime.reroot: with method or node: least-squares

3.41	TreeTime.reroot: rerooting will ignore covariance and shared ancestry.

4.21	###TreeTime.run: INITIAL ROUND

8.05	TreeTime.reroot: with method or node: least-squares

8.05	TreeTime.reroot: rerooting will ignore covariance and shared ancestry.

8.26	###TreeTime.run: rerunning timetree after rerooting

12.25	###TreeTime.run: ITERATION 1 out of 2 iterations

21.78	###TreeTime.run: ITERATION 2 out of 2 iterations

Inferred sequence evolution model (saved as data/sequence_evolution_model.txt):
Substitution rate (mu): 1.0

Equilibrium frequencies (pi_i):
  A: 0.2963
  C: 0.1785
  G: 0.1935
  T: 0.3218
  -: 0.01

Symmetrized rates from j->i (W_ij):
	A	C	G	T	-
  A	0	0.3088	1.3087	0.1327	26.1119
  C	0.3088	0	0.2308	3.5669	16.0944
  G	1.3087	0.2308	0	1.0538	15.8746
  T	0.1327	3.5669	1.0538	0	27.75
  -	26.1119	16.0944	15.8746	27.75	0

Actual rates from j->i (Q_ij):
	A	C	G	T	-
  A	0	0.0915	0.3878	0.0393	7.7378
  C	0.0551	0	0.0412	0.6366	2.8726
  G	0.2532	0.0447	0	0.2039	3.0711
  T	0.0427	1.1477	0.3391	0	8.9289
  -	0.2601	0.1603	0.1581	0.2764	0


Inferred sequence evolution model (saved as data/molecular_clock.txt):
Root-Tip-Regression:
 --rate:	8.000e-04
 --r^2:  	0.11

--- saved tree as 
	 data/timetree.pdf

--- root-to-tip plot saved to  
	data/root_to_tip_regression.pdf

--- alignment including ancestral nodes saved as  
	 data/ancestral_sequences.fasta

--- saved divergence times in 
	 data/dates.tsv

--- tree saved in nexus format as  
	 data/timetree.nexus

--- divergence tree saved in nexus format as  
	 data/divergence_tree.nexus
```

```R
> require(ggfree)
Loading required package: ggfree
Loading required package: ape

Attaching package: ‘ggfree’

The following object is masked from ‘package:ape’:

    unroot

> phy <- read.tree('~/git/covizu/working/timetree.nwk')
> plot(tree.layout(phy))
Warning message:
In as.phyloData(phy, unscaled) :
  as.phyloData ignoring attribute root.edge of length 1
> axis(side=1)
```
![image](https://user-images.githubusercontent.com/1109328/96378077-b0bdfc80-1157-11eb-8a40-2dc95a25704c.png)
